src/components/molecules/OakTimer/OakTimer.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import styled from "styled-components";
+
+import { OakBox, OakBoxProps, OakLabel } from "@/components/atoms";
+
+export type OakTimerProps = {
+  timeCode: string;
+} & Omit<OakBoxProps, "onClick" | "label">;
+
+const StyledTimeBox = styled(OakBox)`
+  opacity: 0.9;
+  padding-left: 2px;
+  padding-right: 2px;
+`;
+
+export const OakTimer = (props: OakTimerProps) => {
+  const { timeCode, ...oakBoxProps } = props;
+
+  function formatTimeCode(timeCode: string): string {
+    const [, minutes, seconds] = timeCode.split(":");
+    return `${minutes}:${seconds}`;
+  }
+
+  const formattedTimeCode = formatTimeCode(timeCode);
+
+  return (
+    <StyledTimeBox
+      $color={"text-inverted"}
+      $borderRadius={"border-radius-xs"}
+      $background={"bg-icon"}
+      $font={["body-4"]}
+      {...oakBoxProps}
+    >
```

Comment:
@k-huggs would it be possible to add something like the below to improve the accessibility for screen readers? currently it just reads the number without any context to what that number is. 
`aria-label={`Media clip length: ${formattedTimeCode}`}`